presto-native-execution/presto_cpp/presto_protocol/connector/iceberg/presto_protocol_iceberg.yml

```diff
@@ -0,0 +1,170 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+EnumMap:
+  AggregationNode:
+    Step: AggregationNodeStep
+
+  ExchangeNode:
+    Type: ExchangeNodeType
+    Scope: ExchangeNodeScope
+
+  LimitNode:
```

Comment:
Same question here. I wonder if all these definitions are needed for connector protocol?